containers/podman-desktop
# feat: display notification on task manager (#4406)

### What does this PR do?

This PR allow to display notifications using the bell icon on the bottom toolbar.

The idea is this one.
As a completed (successfully or not) task can be considered like a notification, we merge the actual notifications with the completed tasks in the bottom of the task manager and keep the running task on the top.
If you click on the `clear notifications` button everything gets deleted except the running tasks.

### Screenshot/screencast of this PR

![notifications_bell](https://github.com/containers/podman-desktop/assets/49404737/ed25737f-f0b6-4415-9104-01855c13609f)

### What issues does this PR fix or reference?

it resolves #4406 

### How to test this PR?

1. be sure to not have any podman machine to see the onboarding notification, then create a new machine or build an image or do something else to display an old task and see it works as before, and notifications + completed task are grouped in the bottom
